Oliver-ke/Teamwork-frontend
# #29535566 Implement admin create user feature

#### What does this PR do?
- Allows admin view and create users

#### Description of Task to be completed?
- add a page for admin to create user
- add a table to display current users

#### How should this be manually tested?
- clone this repository
- run `npm install` to install a dependencies
- run `npm run start` to start the dev server
- login to the dashboard with a valid password and email with admin priviladge
- navigate to users -> add users 
- you should be able to add users

#### Any background context you want to provide?
none

#### What are the relevant project board stories?
[#29535566](https://github.com/Oliver-ke/Teamwork-frontend/projects/1#card-29535566)

#### Screenshots (if appropriate)

#### Questions:
